site/get-started-developing.md

```diff
@@ -39,6 +39,11 @@ Now that we know everything is working with `flux-getting-started`, we're going
     - [Docker](https://docs.docker.com/install/)
     - [Go](https://golang.org/doc/install)
 
+1. Configure your enviroment so you can run tests.  Run:
+    ```sh
+    make test
+    ```
+
 1. We want to make sure we're starting fresh.  Tell Minikube to clear any previously running clusters.
```

Comment:
```suggestion
2. We want to make sure we're starting fresh.  Tell Minikube to clear any previously running clusters.
```

Reply:
all of the bullets are `1.`.  this practice prevents any chance of misnumbering the individual bullets in the future when edits are made.

Reply:
Oh, upon further inspection it looks like all list items start with `1`. Is there a reason behind this?

Reply:
in other words: this is a markdown best-practice I am following.  Markdown ignores your numbering, actually, anyway:

1. this is numbered with a `1`
2. this is numbered with a `2`
4. this is numbered with a `4`
3. this is numbered with a `3`

check the raw text on the above to see what I mean ^

Reply:
Ah, yeah I guess that makes sense as GitHub corrects them.

It would however surprise me as I am a fervent `cat *.md` user.

Reply:
it's not GitHub, this is the spec-compliant behavior of markdown

Reply:
well, the GitHub markdown spec.  I won't get into the various messes with markdown standards.  but, to my knowledge, this is how all markdown parsers work

Reply:
Of course parsers will display it just fine. The whole idea behind Markdown however is that it also reads as well formatted text without parsing.

> Philosophy
> Markdown is intended to be as easy-to-read and easy-to-write as is feasible.
> Readability, however, is emphasized above all else. A Markdown-formatted document should be publishable as-is, as plain text, without looking like it’s been marked up with tags or formatting instructions. 

I do however expect people to read this doc from within GitHub so I suppose ease of use > strict correctness applies.

Reply:
I guess I don't see what's not well formatted.  That's the syntax, right..  It reads well to me.  Maybe that's just because I'm used to it.

Like, to use your quote:
> without looking like it’s been marked up with tags or formatting instructions
to me, specifying `1.`, `2.`, `3.` is a formatting instruction - especially once you consider that that formatting instruction is _ignored_ when the file is rendered.